docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -157,10 +157,18 @@ If you have include files in non-standard places you can pass an
         ext_modules=cythonize("src/*.pyx", include_path=[...]),
     )
 
+To add C/C++ compiler options, e.g. the search path for C header files,
+pass a dictionary object as ``extension_options`` parameter. The dictionary
+values will be appended to the options of the generated Extension
+objects. Check the `distutils documentation`_ for a description of
+possible options.  Some useful options to know about
+are ``include_dirs``, ``libraries``, and ``library_dirs`` which specify where
+to find the ``.h`` and library files when linking to external libraries.
+
 Often, Python packages that offer a C-level API provide a way to find
 the necessary include files, e.g. for NumPy::
 
-    include_path = [numpy.get_include()]
+    extension_options = {'include_path': numpy.get_include()}
```

Comment:
AFAICT, `include_path` is not a distutils option. I believe you meant `include_dirs` in this example.

Reply:
Yes, you are right: https://setuptools.pypa.io/en/latest/userguide/ext_modules.html Thanks!